Tools/machines/perlmutter-nersc/perlmutter_warpx.profile.example

```diff
@@ -48,6 +48,10 @@ export CRAY_ACCEL_TARGET=nvidia80
 # optimize CUDA compilation for A100
 export AMREX_CUDA_ARCH=8.0
 
+# optimize CPU microarchitecture for AMD EPYC 3rd Gen (Milan/Zen3)
+export CXXFLAGS="-march=znver3"
+export CFLAGS="-march=znver3"
```

Comment:
Actually, since #3079 we use the `CC` wrappers, that have these flags.
```suggestion
# optimize CPU microarchitecture for AMD EPYC 3rd Gen (Milan/Zen3)
# note: the cc/CC/ftn wrappers below add those
#export CXXFLAGS="-march=znver3"
#export CFLAGS="-march=znver3"
```

Reply:
Tested via
```
objdump -d ./warpx > warpx.asm 
grep ymm warpx.asm 
```
and it's there :)